### ISSUE TITLE
Color Contrast

### ISSUE THREAD

**Issue by jenniferh2** [teachaccess/tutorial#98]
# Expected Behavior

Header of div.playground-header does not meet 4.5:1 WCAG color contrast guidelines

# Current Behavior

Current contrast is 2.44:1

## Steps to reproduce

Please provide detailed steps for reproducing the issue.

1. Go to module 2.2
2. Look for the gray horizontal bars with text inside
3. Use Adobe Color Analyzer to sample the gray background and the white text and output the contrast ratio

Solutions:
There are many many many color combos that would work. Here are 3.
Gray neutral with white text: background #4E555A (Pepper 100) / text #FFFFFF (Pepper 0)
Blue but not Intuit brand blue with white text: background #1D5696 (Blueberry 90) / text #FFFFFF (Pepper 0)
Warm neutral with black text: #EAEAE3 (Tofu 200) / #000000 (Pepper 140)

**Comment by rachelruderman** [comment]
Thank you Jennifer!! As a designer, which solution is your favorite and what would the hover color be?

**Comment by jenniferh2** [comment]
I'm not used to headers having hover states. Given the prevalence of black text on the page I would chose the Warm neutral with black text.

**Comment by 7mary4** [comment]
The basic is #757575, which is the lightest grey that is accessible with white text. 
--ifm-color-gray-600: #757575;

**Comment by 7mary4** [comment]
![headings with accessible grey background](https://user-images.githubusercontent.com/459953/151714866-baa524f3-5ac2-4b9d-bd8c-f6e4da75afe8.png)
